### PR TITLE
cmd/observe: Add node name filter

### LINKED ISSUE #266: filter by node names
with multi node filtering by node names will become handy

### PR DESCRIPTION
**Pull request by twpayne**
Requires https://github.com/cilium/cilium/pull/13938 to be merged and Cilium to be updated before this will compile.

Closes #266.

### CODE REVIEW

File: cmd/observe/observe.go [hunk 1]
```diff
@@ -211,6 +211,10 @@ more.`,
 		"to-identity", ofilter,
 		"Show all flows terminating at an endpoint with the given security identity"))
 
+	observerCmd.Flags().Var(filterVar(
+		"node-names", ofilter,
+		`Show all flows which match the given node names (e.g. "k8s*", "test-cluster/*.company.com")`)
```
Comment (glibsm) [inline cmd/observe/observe.go:216]: i think this line is missing a closing `)` which is what is making the linter freak out

Comment (twpayne) [reply cmd/observe/observe.go]: Good catch, thanks!

**Comment by twpayne** [comment]
@Rolinh @gandro This is now ready for review.

**Comment by twpayne** [comment]
Fixes #266.

**Review by Rolinh — APPROVED** [review]
Seems to work as expected, brilliant :tada: 
```
$ hubble observe --node-name 'kind-control*' --last 1000 --print-node-name | head
TIMESTAMP             NODE                 SOURCE               DESTINATION          TYPE                                      VERDICT     SUMMARY
Nov 20 13:55:19.001   kind-control-plane   10.244.0.90:46408    10.244.2.63:4240     to-overlay                                FORWARDED   TCP Flags: SYN
Nov 20 13:55:19.720   kind-control-plane   10.244.0.90          10.244.1.236         to-overlay                                FORWARDED   ICMPv4 Redirect(Host)
Nov 20 13:55:19.720   kind-control-plane   10.244.0.90          10.244.1.236         Unsupported protocol for NAT masquerade   DROPPED     ICMPv4 Redirect(Host)
Nov 20 13:55:20.014   kind-control-plane   10.244.0.90:46408    10.244.2.63:4240     to-overlay                                FORWARDED   TCP Flags: SYN
Nov 20 13:55:20.014   kind-control-plane   10.244.0.90:46408    10.244.2.63:4240     to-overlay                                FORWARDED   TCP Flags: ACK
Nov 20 13:55:20.014   kind-control-plane   10.244.0.90:46408    10.244.2.63:4240     to-overlay                                FORWARDED   TCP Flags: ACK, PSH
Nov 20 13:55:20.014   kind-control-plane   10.244.0.90:46408    10.244.2.63:4240     to-overlay                                FORWARDED   TCP Flags: ACK
Nov 20 13:55:20.015   kind-control-plane   10.244.0.90:49010    10.244.3.37:4240     to-overlay                                FORWARDED   TCP Flags: SYN
Nov 20 13:55:20.015   kind-control-plane   10.244.0.90:49010    10.244.3.37:4240     to-overlay                                FORWARDED   TCP Flags: ACK
```

**Review by gandro — APPROVED** [review]

**Pull request merged**